monkestation/code/datums/keybinding/living.dm

```diff
@@ -21,10 +21,17 @@
 		man_radio = locate(/obj/item/radio) in L.contents
 		if(!man_radio)
 			return
-		if(man_radio.channels)
-			channel_options += man_radio.channels
 		if(issilicon(L))
+			if(locate(/obj/item/radio/borg/syndicate) in contents)
+				channel_options -= "General"
```

Comment:
you're removing `"General"` here, is that intentional?

Reply:
Yes.
Syndicate cyborgs do not have access to general.
When they speak over "general", they go straight to the Syndicate channel instead